src/test/java/cosminsechel/tests/tema8/TestPage/BaseTest.java

```diff
@@ -0,0 +1,3 @@
+package cosminsechel.tests.tema8.TestPage;
```

Comment:
Please rename the name of the package from TestPage -> testpage (lowercase for package names as per Java conventions). Thanks!